src/cplus_plugin/main.py

```diff
@@ -504,3 +502,11 @@ def initialize_report_settings():
 
     if settings_manager.get_value(Settings.REPORT_CI_LOGO, None) is None:
         settings_manager.set_value(Settings.REPORT_CI_LOGO, CI_LOGO_PATH)
+
+
+def initialize_zero_value_raster_path():
```

Comment:
We need to include the latest workflow so this initialization can be run each time user installs a new plugin version.

Reply:
This has now been incorporated. In the long term, we might want to consider removing boilerplate code by the caller for checking, removing and updating the settings.